### PR TITLE
fix: added transport.changed event to event map

### PR DESCRIPTION
**Pull request by vishalnarkhede**
## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] Code changes are tested

## Changelog

-

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -1052,50 +1053,7 @@ export type EventHandler<
...
-  | 'reaction.updated'
-  | 'typing.start'
-  | 'typing.stop'
-  | 'user.banned'
-  | 'user.deleted'
-  | 'user.presence.changed'
-  | 'user.unbanned'
-  | 'user.updated'
-  | 'user.watching.start'
-  | 'user.watching.stop';
```
Comment (ferhatelmas) [inline src/types.ts:1098]: this list is bigger than the other

Comment (vishalnarkhede) [reply src/types.ts:1098]: This list is smaller actually than the other. Its missing `channel.kicked` event, which is fixed by this change

Comment (vishalnarkhede) [reply src/types.ts:1098]: This one has 42 types (excluding `all`), and other one has 43.

Comment (ferhatelmas) [reply src/types.ts:1098]: right, it was diff not expanded completely. The place of event might be local ones though

Comment (ferhatelmas) [reply src/types.ts:1098]: otherwise, good reduction in duplication

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Review by AnatolyRugalev — APPROVED** [review]

**Pull request merged**